### PR TITLE
MusED2017 - Blog post writeup

### PR DESCRIPTION
**Pull request by khoeger**
Record the happenings of the MusED 2017 hackathon, including

- overview of theme, partners, schedule
- event related images, video
- general overview of projects

### CODE REVIEW

File: _events/2017-08-05.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Stay tuned to our blog, twitter, or facebook for more announcements!
 - 1:00 PM - Hacking, Brainstorming session, and Workshops start
 - 8:00 PM - Concert / Demos
 
-# Talks
+# Talks
```
Comment (trevorknight) [inline _events/2017-08-05.md]: Please revert this file:

$ git checkout origin/master [filename]

File: _posts/2017-08-05-MusED-Hackathon-2017.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+title:  "MusED Hackathon 2017"
+date:   2017-09-04 12:00:00 -0400
+categories: update
+---
+The [MusED Hackathon]({{site.baseurl}}{% link _events/2017-08-05.md %}), held on August 5, revolved around Music Education, Music Experience Design, and musing about and muses for Music. It was held in collaboration with the [IMPACT Conference](https://impact.musedlab.org/) and NYU Steinhardt's [MusEDLab](https://musedlab.org/) lab.
+
+After kicking off the day with a set of invigorating talks at 12 pm and a pizza break before 2 pm, participants took part in workshops or gathered in groups and created hacks.
+
+![Img: Thoughts from the talks](/assets/events/20170805/20170805_talk4.jpg)
```
Comment (trevorknight) [inline _posts/2017-08-05-MusED-Hackathon-2017.md]: Is it standard to do "Img: "?

The alt text is for accessibility, it generally describes the picture. "Thoughts from the talks" might leave the user wanting more.

File: _posts/2017-08-05-MusED-Hackathon-2017.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+Kushagra Sharma, Manaswi Mishra, and Siddharth Bhardwaj, visiting us from [Music Hackathons Bulgaria](https://www.facebook.com/MusicHackathonsBG/) kicked off the presentations with the project that brough them to NY, Samsara. They presented that, as well as the project they worked on at the hackathon, and were followed by Todd Gregory, Derek Bioh, Liam Baum, Patrick S. Lee, Adam November, Cole Mahlowitz, Jonathan Marmor, Antony Fonville, Tasdique, Sammy, Omar, and Aman. The projects ranged from apps (using music and visualizations and in support of performing artists), to cardboard and aluminum instruments, to a physical version of a beat generation app, to a musical composition created from a video recording.
+
+![Img: Final Presentation Audience](/assets/events/20170805/20170805_finalPresentations.jpg)
+
+We thank all the participants, speakers, volunteers, and our co-sponsors for helping make this a wonderful event.
```
Comment (trevorknight) [inline _posts/2017-08-05-MusED-Hackathon-2017.md]: It seems like several of the images aren't used. Was that your intention?

File: _events/2017-08-05.md (hunk 1 above)
Comment (khoeger) [reply _events/2017-08-05.md]: Good to know, thanks!

File: _posts/2017-08-05-MusED-Hackathon-2017.md (hunk 3 above)
Comment (khoeger) [reply _posts/2017-08-05-MusED-Hackathon-2017.md]: Yes! I couldn't figure out how to create a gallery/tile the images using markdown, or flip through them in place, so decided that perhaps a better way to use them would be to include them in the individual project writeups. I plan on adding them there. 

Do you think it makes more sense to just dump them all at the end of this post?

File: _posts/2017-08-05-MusED-Hackathon-2017.md (hunk 2 above)
Comment (khoeger) [reply _posts/2017-08-05-MusED-Hackathon-2017.md]: I don't know the consensus about doing it in Markdown. Just that it's standard in TeX. It didn't render at first when  I didn't have the Img: but now it's fine, so perhaps something else went wrong and I inadvertently fixed it.

**Comment by khoeger** [comment]
The updates (apart from your suggested fixes) include

- pictures shared with us from MusED Lab and Impact
- individual project write ups

Please let me know what needs changes!

**Review by trevorknight — APPROVED** [review]

**Pull request merged**